packages/react-components/global-context/src/global-context.cy.tsx

```diff
@@ -28,13 +28,13 @@ describe('createContext', () => {
   beforeEach(() => cleanWindowSymbols(SYMBOL_NAMESPACE));
 
   it('should create context on window', () => {
-    const MyContext = createContext({}, 'MyContext', 'package', '9.0.0');
+    const MyContext = createContext(undefined, 'MyContext', 'package', '9.0.0');
     expect(getWindowProperty(getGlobalContextSymbols(SYMBOL_NAMESPACE)[0])).equals(MyContext);
   });
 
   it('should create single context', () => {
-    const MyContext = createContext({}, 'MyContext', 'package', '9.0.0');
-    const MyContext2 = createContext({}, 'MyContext', 'package', '9.0.0');
+    const MyContext = createContext(undefined, 'MyContext', 'package', '9.0.0');
```

Comment:
Fixes linting error: ` error  Invalid default value for context declaration, default value should be undefined  @fluentui/no-context-default-value`